content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -272,16 +275,66 @@ transaction functions. We recommend you get in and out as quick as possible!
 
 </Admonition>
 
-## Limitations
+#### Transaction isolation level
 
-### Transaction Isolation Level
+<Admonition type="info">
 
-Transaction isolation level is not currently configurable at a Prisma level and is not explicitly set by Prisma.
+Prisma does not currently support this feature on MongoDB.
```

Comment:
Is this a temporary limitaton, or does MongoDB just not support isolation levels? Then this wording could be a bit different.

Reply:
Mongo does not support this in general, this is not prisma limitation.

Reply:
Changed the sentence to clarify this